drawee/src/main/java/com/facebook/drawee/drawable/RoundedBitmapDrawable.java

```diff
@@ -270,7 +298,10 @@ private void updateTransform() {
       mIsShaderTransformDirty = true;
       mParentTransform.invert(mInverseParentTransform);
       mTransform.set(mParentTransform);
-      mTransform.preConcat(mBoundsTransform);
+      if (!mBoundsTransform.isIdentity()) {
```

Comment:
Are you sure that this is worth doing? `preConcat` should end up in https://github.com/google/skia/blob/master/src/core/SkMatrix.cpp#L715 which already does the same check internally.